int64 id
string test
string type
string accuracy
